### PR TITLE
 perspective lead wall + smoothwall lead wall

### PR DESCRIPTION
**Pull request by Skeletonman0**
[sprites]
## About the PR 

replaces the top down perspective sprite for lead walls with a 3/4 perspective sprite.

also adds tileable windows instead of the tiny windows at cogwerk's suggestion

## Why's this needed?

humans are 3/4 perspective now and these walls better match that perspective

image

### CODE REVIEW

**Comment by ZeWaka** [comment]
The weird random pixels on the wall are strange, especially given that they differ in size as such. It almost looks like mixels.
It'd be nice if either they were gone or they were more stylized such as the concrete walls seen in Manta.

**Comment by Skeletonman0** [comment]
it seems like the differing size of those pixels is due to being zoomed out in sdmm to get a picture of all the types.

image

now that smoothwall related code is merged and there is a tool to generate the sprites needed for them, i might look into making these into an smoothwall compatible auto-wall instead if that would be better.

**Comment by Skeletonman0** [comment]
smoothwall / autowall support added, really hoping this goes well

**Review by ZeWaka — CHANGES_REQUESTED** [review]
Your .dmm files are not in the TGM format. Please use either the converter or StrongDMM to format them properly.

File: code/turf/turf_autoalign.dm [hunk 1]
```diff
@@ -669,6 +669,46 @@
 		..()
 		setMaterial(getMaterial("coral"))
 
+// lead wall resprite by skeletonman0.... hooray for smoothwalls!
+/turf/unsimulated/wall/auto/leadblue
```
Comment (ZeWaka) [inline code/turf/turf_autoalign.dm:673]: I'd rather these share an abstract ancestor, this is a lot of needless duplication.

**Review by ZeWaka — CHANGES_REQUESTED** [review]

File: code/turf/turf_autoalign.dm (hunk 1 above)
Comment (Skeletonman0) [reply code/turf/turf_autoalign.dm]: fixed

**Comment by Skeletonman0** [comment]
hopefully dealt with a merge conflict in the last change

**Pull request merged**